src/test/java/com/openlattice/rehearsal/datasets/DataSetsTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.openlattice.rehearsal.datasets
+
+import com.openlattice.organization.OrganizationIntegrationAccount
+import com.openlattice.organizations.Organization
+import com.openlattice.rehearsal.authentication.MultipleAuthenticatedUsersBase
+import junit.framework.Assert
+import org.junit.BeforeClass
+import org.junit.Test
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.*
+
+
+open class DataSetsTest : MultipleAuthenticatedUsersBase() {
+    companion object {
+        lateinit var credentials: OrganizationIntegrationAccount
+        lateinit var organization: Organization
+        lateinit var organizationID: UUID
+        lateinit var url: String
+
+        const val create_query = "CREATE TABLE IF NOT EXISTS " +
+                "authors (id serial PRIMARY KEY, name VARCHAR(25));"  +
+                "INSERT INTO authors(id, name) VALUES(1, 'Jack London');"
+        val delete_column_query = "ALTER TABLE authors " +
```

Comment:
*most* minor style things:
- Can these constants be capitalized? In general, we try to CAPITAL_SNAKE_CASE static constants, and camelCase variable names
- you don't need the semicolon at the end of the queries